### PR TITLE
Fix bug 1195533

### PR DESCRIPTION
**Pull request by anivan-suse**
#### Description:
Add bash remediation for rule: file_etc_security_opasswd

#### Rationale:
- Fixes # Bug 1195533: https://bugzilla.suse.com/show_bug.cgi?id=1195533

### CODE REVIEW

File: linux_os/guide/system/permissions/files/permissions_important_account_files/file_etc_security_opasswd/bash/shared.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# platform = multi_platform_sle
```
Comment (matejak) [inline linux_os/guide/system/permissions/files/permissions_important_account_files/file_etc_security_opasswd/bash/shared.sh:1]: I suggest to leave `multi_platform_all` here - the applicability of the remediation is already restricted by the applicability of the rule. The fix doesn't contain anything that is SLE-specific and not part of the rule scope.

File: linux_os/guide/system/permissions/files/permissions_important_account_files/file_etc_security_opasswd/bash/shared.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+# Create /etc/security/opasswd if needed
+# Owner group mode root.root 0600
```
Comment (matejak) [inline linux_os/guide/system/permissions/files/permissions_important_account_files/file_etc_security_opasswd/bash/shared.sh:4]: The remediation code explains itself quite clearly, so I suggest to get rid of those comments.

File: linux_os/guide/system/permissions/files/permissions_important_account_files/file_etc_security_opasswd/bash/shared.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+[ -f  /etc/security/opasswd ] || touch /etc/security/opasswd
+chown root.root /etc/security/opasswd
```
Comment (matejak) [inline linux_os/guide/system/permissions/files/permissions_important_account_files/file_etc_security_opasswd/bash/shared.sh:6]: The user-group separator is `:` already for some time, and [usage of dot is discouraged](https://www.gnu.org/software/coreutils/manual/html_node/chown-invocation.html).

**Review by matejak — CHANGES_REQUESTED** [review]
Thank you for your PR, there are just some minor issues that can be fixed easily, plus a suggestion to remove comments.

**Comment by matejak** [comment]
Thank you for fixing the remediation, it looks good now, but please give us a test scenario as well.

**Comment by brett060102** [comment]
This PR can be closed. The issue was addressed in PR https://github.com/ComplianceAsCode/content/pull/8552

**Pull request closed without merging**